org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/resource/MimeType.java

```diff
@@ -88,20 +95,25 @@ public enum MimeType implements IMimeType {
   THMX("application/vnd.openxmlformats-officedocument.presentationml.presentation", "thmx", IMimeMagic.ZIP),
   TIF("image/tiff", "tif", IMimeMagic.TIF_TIFF),
   TIFF("image/tiff", "tiff", IMimeMagic.TIF_TIFF),
+  TTF("font/ttf", "ttf"),
   TXT("text/plain", "txt"),
   VCARD("text/vcard", "vcard"),
   VCF("text/x-vcard", "vcf"),
   VCS("text/x-vcalendar", "vcs"),
+  WAV("audio/wav", "wav"),
   WEBM("video/webm", "webm"),
-  WOFF("application/font-woff", "woff", IMimeMagic.WOFF),
+  WOFF("font/woff", "woff", IMimeMagic.WOFF),
+  WOFF2("font/woff2", "woff2"),
+  XHTML("application/xhtml+xml", "xhtml"),
   XLS("application/vnd.ms-excel", "xls", IMimeMagic.DOC_XLS_PPT),
   XLSB("application/vnd.ms-excel.sheet.binary.macroEnabled.12", "xlsb"),
   XLSX("application/vnd.openxmlformats-officedocument.spreadsheetml.sheet", "xlsx", IMimeMagic.DOCX_XLSX_PPTX),
   XLSM("application/vnd.ms-excel.sheet.macroEnabled.12", "xlsm", IMimeMagic.DOCX_XLSX_PPTX),
   XLTX("application/vnd.openxmlformats-officedocument.spreadsheetml.template", "xltx", IMimeMagic.DOCX_XLSX_PPTX),
-  XML("text/xml", "xml"),
+  XML("application/xml", "xml"),
```

Comment:
I think `application/xml` is ok (see [RFC 3023](https://datatracker.ietf.org/doc/html/rfc3023#section-3)), but we need to change the tests accordingly.